Description: nall/Makefile fixes

* Use gcc 4.8.
* Use the flags from CFLAGS, CPPFLAGS and LDFLAGS. They are set in the rules file.

Author: Tobias Hansen <thansen@debian.org>

--- a/nall/Makefile
+++ b/nall/Makefile
@@ -44,14 +44,14 @@
     flags := -w -stdlib=libc++
     link := -lc++ -lobjc
   else
-    compiler := g++-4.7
+    compiler := g++-4.8
     flags :=
-    link :=
+    link := $(LDFLAGS)
   endif
 
-  cflags := -x c -std=gnu99
+  cflags := -x c -std=gnu99 $(CFLAGS) $(CPPFLAGS)
   objcflags := -x objective-c -std=gnu99
-  cppflags := -x c++ -std=gnu++11
+  cppflags := -x c++ -std=gnu++11 $(CFLAGS) $(CPPFLAGS)
   objcppflags := -x objective-c++ -std=gnu++11
 endif
 
